static/templates/recipient_row.hbs

```diff
@@ -50,7 +50,11 @@
                     <span class="topic_edit_form" id="{{id}}"></span>
                 </span>
 
+                {{#if topic_muted}}
+                <i class="fa fa-bell on_hover_topic_unmute recipient_bar_icon" data-stream-id="{{stream_id}}" data-topic-name="{{topic}}" title="{{t 'Unmute topic' }}" role="button" tabindex="0" aria-label="{{t 'Mute topic' }} (M)"></i>
```

Comment:
I guess the `aria-label` attribute should be "Unmute" too.
And we should make the same change in `recent_topic_row.hbs`